src/frontend/src/modules/Meter/Transactions.vue

```diff
@@ -92,6 +92,13 @@ export default {
         this.transactions.tokens.length,
       )
     },
+    formatToken(token) {
+      // Ensure token is a string
+      const tokenStr = String(token)
+      // Format in the desired pattern
+      // return tokenStr.match(/.{1,4}/g).join('-'); // For "1234-1234-1234"
+      return tokenStr.match(/.{1,3}/g).join(" ") // For "123 412 341 234"
+    },
```

Comment:
Maybe you can move this into a JS mixing, similar to `currency` and `timing ` where formatting functions like `readable` (for energy) and `timeForTimeZone ` (for timing) are defined?